ManagedCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ManagedCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by kkolontay on 7/13/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension ManagedCache {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
+		return NSFetchRequest<ManagedCache>(entityName: "ManagedCache")
+	}
+
+	@NSManaged public var timestamp: Date
+	@NSManaged public var feed: NSOrderedSet
```

Comment:
These properties should not be public as `ManagedCache` is an internal implementation detail

```suggestion
	@NSManaged var timestamp: Date
	@NSManaged var feed: NSOrderedSet
```

Reply:
done